src/controllers/AuthController.ts

```diff
@@ -60,11 +59,13 @@ export class AuthController extends HTMLController {
     const cookieSuffix = randomBytes(16).toString('base64url')
     const nonce = randomBytes(32).toString('base64url')
     res.cookie(`VERITABLE_NONCE.${cookieSuffix}`, nonce, nonceCookieOpts)
+    this.logger.debug('storing VERITABLE_NONCE cookie %s', cookieSuffix)
```

Comment:
This doesn't include the request id does it?

Reply:
Yes, it does, just tested



```sh
/connection","query":{},"params":{},"remoteAddress":"::1","remotePort":58472,"headers":{}},"res":{"statusCode":302,"headers":{"x-powered-by":"Express","x-request-id":"4d06b45b-17c3-42dd-8bea-c5c72b0ead17","location":"http://localhost:3000/auth/login?path=%2Fconnection","vary":"Accept, Accept-Encoding","content-type":"text/html; charset=utf-8","content-length":"80"}},"responseTime":8,"msg":"request completed"}
{"level":20,"time":1726153935588,"pid":96603,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"4d06b45b-17c3-42dd-8bea-c5c72b0ead17","controller":"/auth","msg":"storing VERITABLE_NONCE cookie 85V7LAo_0N7xhIpf4BxhpA"}
{"level":20,"time":1726153935588,"pid":96603,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"4d06b45b-17c3-42dd-8bea-c5c72b0ead17","controller":"/auth","msg":"storing VERITABLE_REDIRECT cookie 85V7LAo_0N7xhIpf4BxhpA"}
{"level":20,"time":1726153935589,"pid":96603,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"4d06b45b-17c3-42dd-8bea-c5c72b0ead17","controller":"/auth","msg":"login redirect to http://localhost:3080/realms/veritable/protocol/openid-connect/auth?response_type=code&client_id=veritable-ui&redirect_uri=http%3A%2F%2Flocalhost%3A3000%2Fauth%2Fredirect&state=85V7LAo_0N7xhIpf4BxhpA.McOrITYBfKV2RsB8un3gVb8pOJ0uvhslEzRael9563s&scope=openid"}
```

Reply:
Ok, managed to get it working, pushed and when you get a moment please give it a moment 

Health checks as discussed in slack:
```json
5a18e6eb700"}
{"level":20,"time":1726213182306,"pid":62025,"hostname":"F4QVY4N44X.local","name":"veritable-ui","msg":"Credential exchange done event for 41b81c77-7769-496b-9fd6-b5a18e6eb700"}
{"level":20,"time":1726213189914,"pid":62025,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"93105bcf-5e9d-4ffe-922f-a428bfcaf163","msg":"request from Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36 agent"}
{"level":30,"time":1726213191925,"pid":62025,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"93105bcf-5e9d-4ffe-922f-a428bfcaf163","req":{"id":"93105bcf-5e9d-4ffe-922f-a428bfcaf163","method":"GET","url":"/health","query":{},"params":{},"remoteAddress":"::1","remotePort":54142,"headers":{}},"res":{"statusCode":304},"responseTime":2014,"msg":"request completed"}
{"level":20,"time":1726213191928,"pid":62025,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"b9afa2f1-9f59-4175-a64b-88f58f942c10","msg":"request from Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36 agent"}
{"level":30,"time":1726213193930,"pid":62025,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"b9afa2f1-9f59-4175-a64b-88f58f942c10","req":{"id":"b9afa2f1-9f59-4175-a64b-88f58f942c10","method":"GET","url":"/health","query":{},"params":{},"remoteAddress":"::1","remotePort":54142,"headers":{}},"res":{"statusCode":304},"responseTime":2004,"msg":"request completed"}
{"level":20,"time":1726213193933,"pid":62025,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"8c2c3304-4064-40a4-aa5a-964af4c49cbb","msg":"request from Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36 agent"}
{"level":30,"time":1726213195936,"pid":62025,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"8c2c3304-4064-40a4-aa5a-964af4c49cbb","req":{"id":"8c2c3304-4064-40a4-aa5a-964af4c49cbb","method":"GET","url":"/health","query":{},"params":{},"remoteAddress":"::1","remotePort":54142,"headers":{}},"res":{"statusCode":304},"responseTime":2004,"msg":"request completed"}
```

and also htmx polling was duplicating so right now -> 
```json
{"level":20,"time":1726213263775,"pid":62025,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"154c402e-8955-401b-8d08-e8b386325d74","msg":"returning connections page {\"connections\":[{\"company_name\":\"ZEVENTZUK LIMITED\",\"company_number\":\"11111188\",\"status\":\"verified_both\",\"agent_connection_id\":\"3a3bf137-93bf-4e25-b8b0-78d71d227a56\",\"pin_attempt_count\":0,\"pin_tries_remaining_count\":null,\"id\":\"1681347b-83f0-47fe-91ad-0623a0c6bdbf\",\"created_at\":\"2024-09-11T08:46:07.940Z\",\"updated_at\":\"2024-09-11T08:48:37.548Z\"},{\"company_name\":\"TECHDIGITAL LIMITED\",\"company_number\":\"11111114\",\"status\":\"pending\",\"agent_connection_id\":null,\"pin_attempt_count\":0,\"pin_tries_remaining_count\":null,\"id\":\"c6492392-50ea-4551-994a-6520ef2a05bc\",\"created_at\":\"2024-09-10T17:48:29.099Z\",\"updated_at\":\"2024-09-10T17:48:29.099Z\"},{\"company_name\":\"DIGITAL CATAPULT\",\"company_number\":\"07964699\",\"status\":\"unverified\",\"agent_connection_id\":\"2ed1b353-fea7-4978-a31b-0c3129aa372f\",\"pin_attempt_count\":0,\"pin_tries_remaining_count\":null,\"id\":\"ed92750e-e158-4960-9cea-3290129f6f5e\",\"created_at\":\"2024-09-04T11:32:52.718Z\",\"updated_at\":\"2024-09-06T07:31:17.219Z\"}],\"search\":\"\"}"}
{"level":30,"time":1726213263778,"pid":62025,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"154c402e-8955-401b-8d08-e8b386325d74","req":{"id":"154c402e-8955-401b-8d08-e8b386325d74","method":"GET","url":"/connection?search=","query":{"search":""},"params":{},"remoteAddress":"::1","remotePort":54179,"headers":{}},"res":{"statusCode":200},"responseTime":10,"msg":"request completed"}
{"level":20,"time":1726213273772,"pid":62025,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"e6a2c655-9d13-46c1-af95-b283e79ebc05","msg":"returning connections page {\"connections\":[{\"company_name\":\"ZEVENTZUK LIMITED\",\"company_number\":\"11111188\",\"status\":\"verified_both\",\"agent_connection_id\":\"3a3bf137-93bf-4e25-b8b0-78d71d227a56\",\"pin_attempt_count\":0,\"pin_tries_remaining_count\":null,\"id\":\"1681347b-83f0-47fe-91ad-0623a0c6bdbf\",\"created_at\":\"2024-09-11T08:46:07.940Z\",\"updated_at\":\"2024-09-11T08:48:37.548Z\"},{\"company_name\":\"TECHDIGITAL LIMITED\",\"company_number\":\"11111114\",\"status\":\"pending\",\"agent_connection_id\":null,\"pin_attempt_count\":0,\"pin_tries_remaining_count\":null,\"id\":\"c6492392-50ea-4551-994a-6520ef2a05bc\",\"created_at\":\"2024-09-10T17:48:29.099Z\",\"updated_at\":\"2024-09-10T17:48:29.099Z\"},{\"company_name\":\"DIGITAL CATAPULT\",\"company_number\":\"07964699\",\"status\":\"unverified\",\"agent_connection_id\":\"2ed1b353-fea7-4978-a31b-0c3129aa372f\",\"pin_attempt_count\":0,\"pin_tries_remaining_count\":null,\"id\":\"ed92750e-e158-4960-9cea-3290129f6f5e\",\"created_at\":\"2024-09-04T11:32:52.718Z\",\"updated_at\":\"2024-09-06T07:31:17.219Z\"}],\"search\":\"\"}"}
{"level":30,"time":1726213273774,"pid":62025,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"e6a2c655-9d13-46c1-af95-b283e79ebc05","req":{"id":"e6a2c655-9d13-46c1-af95-b283e79ebc05","method":"GET","url":"/connection?search=","query":{"search":""},"params":{},"remoteAddress":"::1","remotePort":54181,"headers":{}},"res":{"statusCode":200},"responseTime":6,"msg":"request completed"}
{"level":20,"time":1726213283773,"pid":62025,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"c04f8fe0-fe28-4ec2-970b-fba7b5d2af1b","msg":"returning connections page {\"connections\":[{\"company_name\":\"ZEVENTZUK LIMITED\",\"company_number\":\"11111188\",\"status\":\"verified_both\",\"agent_connection_id\":\"3a3bf137-93bf-4e25-b8b0-78d71d227a56\",\"pin_attempt_count\":0,\"pin_tries_remaining_count\":null,\"id\":\"1681347b-83f0-47fe-91ad-0623a0c6bdbf\",\"created_at\":\"2024-09-11T08:46:07.940Z\",\"updated_at\":\"2024-09-11T08:48:37.548Z\"},{\"company_name\":\"TECHDIGITAL LIMITED\",\"company_number\":\"11111114\",\"status\":\"pending\",\"agent_connection_id\":null,\"pin_attempt_count\":0,\"pin_tries_remaining_count\":null,\"id\":\"c6492392-50ea-4551-994a-6520ef2a05bc\",\"created_at\":\"2024-09-10T17:48:29.099Z\",\"updated_at\":\"2024-09-10T17:48:29.099Z\"},{\"company_name\":\"DIGITAL CATAPULT\",\"company_number\":\"07964699\",\"status\":\"unverified\",\"agent_connection_id\":\"2ed1b353-fea7-4978-a31b-0c3129aa372f\",\"pin_attempt_count\":0,\"pin_tries_remaining_count\":null,\"id\":\"ed92750e-e158-4960-9cea-3290129f6f5e\",\"created_at\":\"2024-09-04T11:32:52.718Z\",\"updated_at\":\"2024-09-06T07:31:17.219Z\"}],\"search\":\"\"}"}
{"level":30,"time":1726213283775,"pid":62025,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"c04f8fe0-fe28-4ec2-970b-fba7b5d2af1b","req":{"id":"c04f8fe0-fe28-4ec2-970b-fba7b5d2af1b","method":"GET","url":"/connection?search=","query":{"search":""},"params":{},"remoteAddress":"::1","remotePort":54182,"headers":{}},"res":{"statusCode":200},"responseTime":5,"msg":"request completed"}
{"level":20,"time":1726213293780,"pid":62025,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"59a9408c-3883-4108-8f00-c2ccd61db047","msg":"returning connections page {\"connections\":[{\"company_name\":\"ZEVENTZUK LIMITED\",\"company_number\":\"11111188\",\"status\":\"verified_both\",\"agent_connection_id\":\"3a3bf137-93bf-4e25-b8b0-78d71d227a56\",\"pin_attempt_count\":0,\"pin_tries_remaining_count\":null,\"id\":\"1681347b-83f0-47fe-91ad-0623a0c6bdbf\",\"created_at\":\"2024-09-11T08:46:07.940Z\",\"updated_at\":\"2024-09-11T08:48:37.548Z\"},{\"company_name\":\"TECHDIGITAL LIMITED\",\"company_number\":\"11111114\",\"status\":\"pending\",\"agent_connection_id\":null,\"pin_attempt_count\":0,\"pin_tries_remaining_count\":null,\"id\":\"c6492392-50ea-4551-994a-6520ef2a05bc\",\"created_at\":\"2024-09-10T17:48:29.099Z\",\"updated_at\":\"2024-09-10T17:48:29.099Z\"},{\"company_name\":\"DIGITAL CATAPULT\",\"company_number\":\"07964699\",\"status\":\"unverified\",\"agent_connection_id\":\"2ed1b353-fea7-4978-a31b-0c3129aa372f\",\"pin_attempt_count\":0,\"pin_tries_remaining_count\":null,\"id\":\"ed92750e-e158-4960-9cea-3290129f6f5e\",\"created_at\":\"2024-09-04T11:32:52.718Z\",\"updated_at\":\"2024-09-06T07:31:17.219Z\"}],\"search\":\"\"}"}
{"level":30,"time":1726213293782,"pid":62025,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"59a9408c-3883-4108-8f00-c2ccd61db047","req":{"id":"59a9408c-3883-4108-8f00-c2ccd61db047","method":"GET","url":"/connection?search=","query":{"search":""},"params":{},"remoteAddress":"::1","remotePort":54186,"headers":{}},"res":{"statusCode":200},"responseTime":7,"msg":"request completed"}
{"level":20,"time":1726213304447,"pid":62025,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"b6708433-6a32-4d95-9d99-3bef5275f31f","msg":"returning connections page {\"connections\":[{\"company_name\":\"ZEVENTZUK LIMITED\",\"company_number\":\"11111188\",\"status\":\"verified_both\",\"agent_connection_id\":\"3a3bf137-93bf-4e25-b8b0-78d71d227a56\",\"pin_attempt_count\":0,\"pin_tries_remaining_count\":null,\"id\":\"1681347b-83f0-47fe-91ad-0623a0c6bdbf\",\"created_at\":\"2024-09-11T08:46:07.940Z\",\"updated_at\":\"2024-09-11T08:48:37.548Z\"},{\"company_name\":\"TECHDIGITAL LIMITED\",\"company_number\":\"11111114\",\"status\":\"pending\",\"agent_connection_id\":null,\"pin_attempt_count\":0,\"pin_tries_remaining_count\":null,\"id\":\"c6492392-50ea-4551-994a-6520ef2a05bc\",\"created_at\":\"2024-09-10T17:48:29.099Z\",\"updated_at\":\"2024-09-10T17:48:29.099Z\"},{\"company_name\":\"DIGITAL CATAPULT\",\"company_number\":\"07964699\",\"status\":\"unverified\",\"agent_connection_id\":\"2ed1b353-fea7-4978-a31b-0c3129aa372f\",\"pin_attempt_count\":0,\"pin_tries_remaining_count\":null,\"id\":\"ed92750e-e158-4960-9cea-3290129f6f5e\",\"created_at\":\"2024-09-04T11:32:52.718Z\",\"updated_at\":\"2024-09-06T07:31:17.219Z\"}],\"search\":\"\"}"}
{"level":30,"time":1726213304449,"pid":62025,"hostname":"F4QVY4N44X.local","name":"veritable-ui","req_id":"b6708433-6a32-4d95-9d99-3bef5275f31f","req":{"id":"b6708433-6a32-4d95-9d99-3bef5275f31f","method":"GET","url":"/connection?search=","query":{"search":""},"params":{},"remoteAddress":"::1","remotePort":54197,"headers":{}},"res":{"statusCode":200},"responseTime":9,"msg":"request completed"}
```